src/content/topics/integrations/more/compass.mdx

```diff
@@ -0,0 +1,144 @@
+---
+path: /integrations/compass
+title: Compass
+description: This topic explains visualize LaunchDarkly feature flag changes in Atlassian Compass.
```

Comment:
```suggestion
description: This topic explains how to use the Compass integration for LaunchDarkly.
```
The description should match the first sentence in the Overview section. Feel free to include something about visualizing feature flag changes if you'd like that in both places.